package.json

```diff
@@ -51,8 +51,8 @@
     "oav": "./cli.js"
   },
   "scripts": {
-    "jshint": "jshint index.js --reporter=jslint",
-    "test": "npm -s run-script jshint && mocha -t 100000",
+    "jshint": "jshint lib --reporter=jslint",
+    "test": "mocha -t 100000",
```

Comment:
does npm run jshint by itself, now? Previously it did not do that. Hence we had to && the jshint script along with mocha.

Reply:
Yup it does not run jshint by itself. https://travis-ci.org/Azure/oav/builds/335452396#L970
So please do the following:
`"test": "npm -s run-script jshint && mocha -t 100000",`

Reply:
@amarzavery  left it out because there as still a couple of linter issues which I was not sure to resolve. Let me look if I can get it clean and then I can add it back

Reply:
Ah I see. I thought you fixed all of them. No worries, if you cannot fix all of them right now.

Reply:
@amarzavery  Getting there lets clean it up and keep it like that :)

any idea why we iterate here, doesn't seem we do anything different in each iteration: https://github.com/Azure/oav/blob/4459e377b6edb95d1d361b1888af186762c0d3dc/lib/xMsExampleExtractor.js#L192

Reply:
you wont be able to replace this with a `for..of`. We are iterating over the responses dictionary in swagger for a given operation to extract examples for every response code
```json5
"responses": {
    "200": {},
    "202: {}
 . . 
}
```